### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AlexeyAbdulmanov**
Added gem responders

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -17,39 +17,25 @@ def edit
...
     @post = current_user.posts.create(post_params)
-
-    respond_to do |format|
-      if @post.save
-        format.html { redirect_to @post, notice: 'Post was successfully created.' }
-      else
-        format.html { render :new }
-      end
-    end
+    respond_with(@post)
```
Comment (ghost) [inline app/controllers/posts_controller.rb]: можно без скобок

Comment (AlexeyAbdulmanov) [reply app/controllers/posts_controller.rb]: ага, в одном месте забыл)

**Review by ghost — APPROVED** [review]

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -57,6 +57,10 @@ and (min-resolution: 192dpi) {
     float: right;
 }
 
+.flash {
```
Comment (ghost) [inline app/assets/stylesheets/application.css:60]: It's better to add bootstrap class like "pull-right" into dev that contains flash messages.

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -36,10 +36,11 @@ html[lang="en"]
               = link_to 'Зарегистрироваться', new_user_registration_path, class: 'btn btn-success btn-outline'
     .container
       .row
-        .col-lg-11
-          = bootstrap_flash
+        .col-lg-8
+          .flash
```
Comment (ghost) [inline app/views/layouts/application.html.slim:40]: .flash.pull-right 

I think it will work

**Review by ghost — APPROVED** [review]
Lets merge

**Pull request merged**